engine/src/main/java/org/terasology/engine/rendering/nui/layers/ingame/metrics/RunningThreadsMode.java

```diff
@@ -35,4 +52,9 @@ public boolean isAvailable() {
     public boolean isPerformanceManagerMode() {
         return false;
     }
+
+    @Override
+    public void close() throws Exception {
+
+    }
```

Comment:
Did you intend to put something here? I see you added an `implements AutoCloseable` to the class.

Reply:
opps was intending to have a temporary registry.